tests/integration/components/select-light-test.js

```diff
@@ -153,8 +153,9 @@ module('Integration | Component | select-light', function(hooks) {
       <SelectLight
         @options={{this.options}}
         @value={{this.value}}
-        @valueKey="val"
-        @displayKey="description" />
+        @valueKey="val">
+				<:option as |optionValue| >{{optionValue.description}}</:option>
```

Comment:
With this new style, it seems more like we are yielding the entire option and not just the option value. So perhaps this is more appropriate.
```suggestion
				<:option as |currentOption| >{{currentOption.description}}</:option>
```